likeHack.js

```diff
@@ -0,0 +1,111 @@
+import mongoose from "mongoose";
+
+const uri = `mongodb://${process.env.MONGO_USER}:${process.env.MONGO_PASSWORD}@ds157136.mlab.com:57136/hackone`;
+let conn = null;
+
+export const like = async (event, context) => {
+  //need a hackid
+  //need a userid
+  //keep an array of likes in hack collecton's document
+  //when user likes put in the likes array
+  //when user unlike, take user out of the likes array
+  context.callbackWaitsForEmptyEventLoop = false;
+  const data = JSON.parse(event.body);
+  const { hackId, userId } = data;
```

Comment:
while the risk and impact is low, by allowing the user to specify a userId, I could like the hack for another user. We're also not validating the user code actually exists, so if i wanted 1 million likes on my hack (maybe to entice people to join it) I could just invent userIds

Reply:
Just wondering would this be mitigated if can extra userId from the jwt token? thank you

Reply:
Yes, we validate the token, so we can trust it is the user with the token.

Reply:
The route is guarded by checkAuth.js middleware, has implemented validation to see if userId is valid from the database. Still working on trying to extract userId from jwt token